### DISCUSSION TITLE
请问是否具有变量的功能？

### DISCUSSION THREAD

**Discussion by bbcvc** [illacloud/illa-builder#713] [Q&A] [answered]
例如：

1. 定义一个input输入框名为input1
2. 在其它组件如何获得input1中输入的值？

这种上下文如果不依赖变量，那么应该如何传递呢？

**Comment by karminski** [comment, marked as answer]
@bbcvc hi, 只需要使用 {{displayName.value}} 即可获得组件内部的值. 比如想要获取 input1 的值, 使用 {{input1.value}} 即可.
